relayer/src/types.ts

```diff
@@ -24,3 +26,10 @@ export interface ParaHeadAndId {
 export type ParachainsMap = Map<string, Parachain>;
 
 export type Brand<K, T> = K & { __brand: T; };
+
+export interface ParachainConfigType {
+  url: string;
+  paraId: number;
+  // TODO: get chain name from api
+  chain: string;
```

Comment:
Shouldn't this type be `ChainName` instead?

Reply:
this is the type for parachain object from config, where name is currently hardcoded, I can add ` as ChainName` as well, but we're going to fetch name from RPC API in the future, so that will change

Reply:
Even if you fetch it from RPC API, it can still be `ChainName` if it has to be.